spec/views/views_test/user_show_spec.rb

```diff
@@ -0,0 +1,140 @@
+require 'rails_helper'
+# rubocop:disable Metrics/BlockLength
+RSpec.feature 'Logins', type: :feature do
+  background { visit new_user_session_path }
+
```

Comment:
Please add a test that asserts the user can see the `user's profile picture`.
Please add a test that asserts the user can see the `user's username`.
Please add a test that redirects to the `post's show page` when you click on a `user's post`.
Please add a test that redirects to the `user's post's index page` when the user clicks on `see all posts`.

Reply:
Hello @SajjadAhmad14 ,
I implemented all your requests in the `user_index_spec.rb` file. kindly, check all my scenarios and you'll find those tests.
code line 5: scenario for `user can see the user's username.`
code line 19: scenario for `user can see the user's profile picture`
. ...
. ...
. ...
Please, kindly check in my `user_index_spec.rb`  file to see all those tests.